lib/granite/projector/translations.rb

```diff
@@ -8,7 +8,8 @@ class TranslationsWrapper
       end
 
       def translate(*args)
-        TranslationsWrapper.new.translate(*self.class.scope_translation_args_by_projector(args))
+        key, options = self.class.scope_translation_args_by_projector(args)
```

Comment:
This is worse, what if amount of arguments changes? Seems very likely to fail in the future

Reply:
Then I suppose we should change the code? Both `TranslationsWrapper` & `.scope_translation_args_by_projector` is inside granite, so I see no issues here.
I can do something like `*wrapper_args, wrapper_options = self.class.scope_translation_args_by_projector(args)` to make it kinda future-proof, but IMHO it hurts readability.

Reply:
Yes let's do that. I think we could merge `scope_translation_args_by_projector` into `translate` and keep them in one module which can then be included in projector & view helpers.

Reply:
Specs are passing in linked PR for the current granite version without code changes [except minor changes in monkeypatching], so I thinks it's ready for review.

Reply:
Ok, now it's ready.